docs/modules/integrate/pages/vertx-step-by-step-guide.adoc

```diff
@@ -0,0 +1,476 @@
+= Get started with Vert.x
+
+This is a step-by-step guide on how to integrate Vert.x with Hazelcast and use Hazelcast for distributed session management and other distributed data structures.
+
+In this guide we will
+- start with a simple Vert.x Hello World application
+- add vertx-hazelcast module and enable distributed session management
+- use `io.vertx.core.shareddata.Counter` data structure to implement a unique id generator
+
+== Prerequisites
+
+- Java 17 or newer
+- Maven 3.9+
+- link:https://httpie.io[httpie] client
+
+== Create a new project
+
+1. Go to link:https://start.vertx.io[start.vertx.io], change the artifact id to `messages`, the version to 5.0.0, and generate a new project.
+
+2. Extract the project and build it using:
+
+[source,bash]
+----
+$ mvn clean package
+----
+
+and start the application using:
+
+[source,bash]
+----
+java -jar target/messages-1.0.0-SNAPSHOT-fat.jar
+----
+
+you should see an output similar to the following:
+
+[source]
+----
+HTTP server started on port 8888
+Aug 29, 2024 2:22:38 PM io.vertx.launcher.application.VertxApplication
+INFO: Succeeded in deploying verticle
+----
+
+[NOTE]
```

Comment:
Is the following 'ToDo' content hidden?

Reply:
This issue has been fixed. Removed the TODO in a823ea70